source/MRMesh/MRExampleTest.cpp

```diff
@@ -0,0 +1,48 @@
+#include "MRMesh/MRMesh.h"
+#include "MRMesh/MRTorus.h"
+#include "MRMesh/MRMeshLoad.h"
+#include "MRMesh/MRMeshRelax.h"
+#include "MRMesh/MRMeshSubdivide.h"
+#include "MRMesh/MRAffineXf3.h"
+#include "MRMesh/MRGTest.h"
+#include "MRCube.h"
+
+namespace MR
+{
+
+TEST( MRMesh, BasicExample )
+{
+    // load mesh
```

Comment:
Please write here in the comments the link to Doxygen file, which needs to be updated with the test